OpenSim/Simulation/Test/testManager.cpp

```diff
@@ -136,3 +162,138 @@ void testStationCalcWithManager()
+    state.setTime(integInitTimes[0]);
+    size_t n = integFinalTimes.size();
+
+    for (size_t i = 0; i < n; ++i) {
+        // Set initial state for integration and check that it's correct
+        sliderCoord.setValue(state, initHeights[i]);
+        sliderCoord.setSpeedValue(state, initSpeeds[i]);
+
+        Manager manager(model);
+        manager.initialize(state);
+        const SimTK::State& initState = manager.getState();
+
+        SimTK_TEST_EQ(initState.getTime(), integInitTimes[i]);
+        SimTK_TEST_EQ(sliderCoord.getValue(initState), initHeights[i]);
+        SimTK_TEST_EQ(sliderCoord.getSpeedValue(initState), initSpeeds[i]);
+
+        // Use Station to get the location, velocity & acceleration in ground.
+        double stationHeight = myStation->getLocationInGround(initState)[1];
+        double stationSpeed = myStation->getVelocityInGround(initState)[1];
+
+        SimTK_TEST_EQ(stationHeight, initHeights[i]);
+        SimTK_TEST_EQ(stationSpeed, initSpeeds[i]);
+
+        state = manager.integrate(integFinalTimes[i]);
+
+        model.realizeVelocity(state);
+
+        double duration = integFinalTimes[i] - integInitTimes[i];
+        double finalHeight = 
+            initHeights[i] + initSpeeds[i]*duration - 0.5*g*duration*duration;
+        double finalSpeed = initSpeeds[i] - g*duration;
+        double sliderHeight = sliderCoord.getValue(state);
+        double sliderSpeed = sliderCoord.getSpeedValue(state);
+        cout << "Slider: t = " << state.getTime() << ", h = " << sliderHeight
+            << ", v = " << sliderSpeed << " | Eq: t = " << integFinalTimes[i]
+            << ", h = " << finalHeight << ", v = " << finalSpeed;
+
+        SimTK_TEST_EQ(state.getTime(), integFinalTimes[i]);
+        SimTK_TEST_EQ(sliderHeight, finalHeight);
+        SimTK_TEST_EQ(sliderSpeed, finalSpeed);
+
+        // Use Station to get the location, velocity & acceleration in ground.
+        stationHeight = myStation->getLocationInGround(state)[1];
+        stationSpeed = myStation->getVelocityInGround(state)[1];
+
+        cout << " | Station: t = " << state.getTime() << ", h = " 
+            << stationHeight << ", v = " << stationSpeed << endl;
+
+        // Compare Station values with equation values
+        SimTK_TEST_EQ(stationHeight, finalHeight);
+        SimTK_TEST_EQ(stationSpeed, finalSpeed);
+    }
+
+}
+
+void testExcitationUpdatesWithManager()
+{
+    cout << "Running testExcitationUpdatesWithManager" << endl;
+    LoadOpenSimLibrary("osimActuators");
+    Model arm("arm26.osim");
+
+    const Set<Muscle> &muscleSet = arm.getMuscles();
+    PrescribedController* controller = new PrescribedController();
+    controller->addActuator(muscleSet.get(0));
+    Constant* fn = new Constant(0);
+    controller->prescribeControlForActuator(0, fn);
+    arm.addController(controller);
+
+    SimTK::State& state = arm.initSystem();
+    state.setTime(0);
+    double stepsize = 0.01;
+    muscleSet.get(0).setActivation(state, 0.1);
```

Comment:
How about if you make set `initAct = 0.1` here and then call `setActivation(state, initAct)` in the loop before initializing the `Manager`? That would also test that you could update the activation state (like  you did with Coordinates previously).